app/controllers/organizations/pets_controller.rb

```diff
@@ -93,4 +94,8 @@ def set_nav_tabs
       {name: "Summary", path: pet_path(@pet)}
     ]
   end
+
+  def determine_active_tab # from 265
```

Comment:
If this is duplicate code in the controller we could pull this into a concern and then multiple controllers can include it to use the logic.

Reply:
I didn't realize this method would be reused. I'll move it.